src/main/resources/assets/securitycraft/lang/fr_fr.json

```diff
@@ -0,0 +1,767 @@
+	"block.securitycraft.reinforced_mossy_cobblestone_slab": "Dalle de Pierre moussue blindée",
+	"block.securitycraft.reinforced_end_stone_brick_slab": "Dalle de Pierre de l’end taillée blindée",
+	"block.securitycraft.reinforced_smooth_sandstone_slab": "Dalle de grès lisse blindée",
+	"block.securitycraft.reinforced_smooth_quartz_slab": "Dalle de Quartz lisse blindée",
+	"block.securitycraft.reinforced_granite_slab": "Dalle de Granite blindée",
+	"block.securitycraft.reinforced_andesite_slab": "Dalle d’Andesite blindée",
+	"block.securitycraft.reinforced_red_nether_brick_slab": "Dalle de Brique rouge du Nether blindée",
+	"block.securitycraft.reinforced_polished_andesite_slab": "Dalle d‘Andésite Polie blindée",
+	"block.securitycraft.reinforced_diorite_slab": "Dalle de Diorite blindée",
+	"block.securitycraft.reinforced_normal_stone_slab": "Dalle de pierre blindée",
+	"block.securitycraft.trophy_system": "Système Trophy",
+	"block.securitycraft.crystal_quartz": "Bloc de Cristal de Quartz",
+	"block.securitycraft.chiseled_crystal_quartz": "Bloc de cristal de Quartz sculpté",
+	"block.securitycraft.crystal_quartz_pillar": "Pilier en cristal de Quartz",
+	"block.securitycraft.reinforced_crystal_quartz_block": "Bloc de cristal de Quartz blindé",
+	"block.securitycraft.reinforced_chiseled_crystal_quartz_block": "Bloc de cristal de Quartz sculpté blindé ",
+	"block.securitycraft.reinforced_crystal_quartz_pillar": "Pilier en cristal de Quartz blindé",
+	"block.securitycraft.crystal_quartz_slab": "Dalle en cristal de Quartz",
+	"block.securitycraft.reinforced_crystal_quartz_slab": "Dalle en cristal de Quartz blindée",
+	"block.securitycraft.crystal_quartz_stairs": "Escalier en cristal de Quartz",
+	"block.securitycraft.reinforced_crystal_quartz_stairs": "Escalier en cristal de Quartz blindé",
+	"block.securitycraft.block_pocket_wall": "Bloc Mur de poche",
+	"block.securitycraft.block_pocket_manager": "Gestionnaire de Bloc de poche",
+	"block.securitycraft.reinforced_stone_pressure_plate": "Plaque de pression en pierre blindée",
+	"block.securitycraft.reinforced_oak_pressure_plate": "Plaque de pression en chêne blindée",
+	"block.securitycraft.reinforced_spruce_pressure_plate": "Plaque de pression en épicéa blindée",
+	"block.securitycraft.reinforced_birch_pressure_plate": "Plaque de pression en bouleau blindée",
+	"block.securitycraft.reinforced_jungle_pressure_plate": "Plaque de pression en acajou blindée",
+	"block.securitycraft.reinforced_acacia_pressure_plate": "Plaque de pression en acacia blindée",
+	"block.securitycraft.reinforced_dark_oak_pressure_plate": "Plaque de pression en chêne noir blindée",
+	"block.securitycraft.reinforced_red_sand": "Sable rouge blindé",
+	"block.securitycraft.reinforced_bookshelf": "Etagère blindée",
+	"block.securitycraft.reinforced_redstone_block": "Bloc de redstone blindé",
+
+	"item.securitycraft.casseur de code": "Décrypteur",
```

Comment:
You accidentally changed the language key here. It should be `item.securitycraft.codebreaker`.

Reply:
Still needs to be changed.